### PR TITLE
Added CONTRIBUTING.md

### LINKED ISSUE #11: Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Saif807380**
Added the guidelines for contributing to the project.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- For developers, please make sure your environment is setup correctly.
+
+## Workflow for submitting a PR
+
+1. Fork the repository to your own GitHub account
+
+2. Clone from your repository
+
+```bash
+    $ git clone https://github.com/your-username/repository-name
```
Comment (gasharova) [inline CONTRIBUTING.md]: change this to contain the repo url

Comment (Saif807380) [reply CONTRIBUTING.md]: This URL will the forked copy right? So we won't know what the username would be.

Comment (gasharova) [reply CONTRIBUTING.md]: the repo name will be the same though

**Review by gasharova — APPROVED** [review]

**Review by gmelodie — APPROVED** [review]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (Saif807380) [reply CONTRIBUTING.md]: I'll update the repository name assuming that's not going to change 🤭

**Pull request merged**